pkg/ipam/subnet.go

```diff
@@ -241,6 +253,13 @@ func (subnet *Subnet) getV4RandomAddress(podName, nicName string, mac string, sk
 	if !part2.Start.GreaterThan(part2.End) {
 		subnet.V4FreeIPList = append(subnet.V4FreeIPList, part2)
 	}
+	if split, NewV4AvailIPRangeList := splitIPRangeList(subnet.V4AvailIPList, ip); split {
+		subnet.V4AvailIPList = NewV4AvailIPRangeList
```

Comment:
can V4FreeIPList used instead of V4AvailIPList ?

Reply:
it could not do this, because V4AvailIPList = V4FreeIPList  + V4ReleaseIPList